dependencies.py

```diff
@@ -0,0 +1,27 @@
+import subprocess
+import pandas as pd
+import json
+
+if __name__ == "__main__":
+    rc = subprocess.call("./dependencies.sh")
```

Comment:
bit dangerous to use subprocess with just an arbitrary shell script, all sorts of nasty things could be put in there.

Reply:
Yeah, from our conversation earlier today, we should just run dependencies.sh separately, and then feed the json outputs into the analysis script